lib/graphql/schema.ts

```diff
@@ -1355,13 +1332,44 @@ export type Invoice = {
 
 export type InvoiceProductOutput = {
   __typename?: 'InvoiceProductOutput';
-  id: Scalars['String'];
+  id?: Maybe<Scalars['String']>;
   description?: Maybe<Scalars['String']>;
   price?: Maybe<Scalars['Float']>;
   vat?: Maybe<Scalars['String']>;
   quantity?: Maybe<Scalars['Float']>;
 };
 
+export type InvoicingDashboardData = {
+  __typename?: 'InvoicingDashboardData';
+  pageInfo: InvoicePageInfo;
+  data: Array<DashboardInvoice>;
+};
+
+export type InvoicePageInfo = {
+  __typename?: 'InvoicePageInfo';
+  hasNextPage: Scalars['Boolean'];
+  hasPreviousPage: Scalars['Boolean'];
+  currentPage: Scalars['Int'];
+};
+
+export type DashboardInvoice = {
+  __typename?: 'DashboardInvoice';
+  id: Scalars['ID'];
+  status: InvoiceStatusType;
+  invoiceNumber?: Maybe<Scalars['Int']>;
+  dueDate?: Maybe<Scalars['DateTime']>;
+  paidAt?: Maybe<Scalars['DateTime']>;
+  amount?: Maybe<Scalars['Int']>;
```

Comment:
Should it rather be `DECIMAL`? the total amount of the invoice may decimal most of the time, like shown on the mockup:
![Screenshot 2021-04-13 at 14 52 21](https://user-images.githubusercontent.com/25295489/114555317-dc81f100-9c67-11eb-8204-3747fc82a79e.png)

Reply:
We always store amounts in cents

Reply:
Right, easier to make calculation in cents 👍